index.bs

```diff
@@ -329,6 +367,42 @@ band communication channel between devices.
 
 Sensors can potentially be used in cross-device linking and tracking of a user.
 
+<h3 id="main-privacy-security-threats">Types of privacy and security threats</h3>
+
+<em>This section is non-normative.</em>
+
+<h4 id="location-tracking">Location Tracking</h4>
+
+Under this type of threat, the attacks use sensor data to locate the device without using
+GPS or any other location sensors. For example, accelerometer data can be used to infer
+the location of smartphones by using statistical models to obtain estimated trajectory,
+then map matching algorithms can be used to obtain predicted location points (within a
+200-m radius)[[MOBILESENSORS]].
+
+<h4 id="eavesdropping">Eavesdropping</h4>
+
+Recovering speech from gyroscope readings is an example of eavesdropping attack.
+See [[GYROSPEECHRECOGNITION]].
+
+<h4 id="keystroke-monitoring">Keystroke Monitoring</h4>
+
+Many user inputs can be inferred from sensor data, this includes a wide range of attacks on
+user PINs, passwords, and lock patterns (and even touch actions such as click, scroll, and
+zoom) using motion sensors. These attacks normally train a machine learning algorithm to
+discover such information about the users. See [[STEALINGPINSVIASENSORS]].
+
+<h4 id="device-fingerprinting">Device Fingerprinting</h4>
+
+Sensors can provide information that can uniquely identify the device using those sensors.
```

Comment:
This is a quotation from "Mobile Sensors and Context-Aware Computing", but I can rephrase..